doc/doxygen/scripts/steps.pl

```diff
@@ -1,6 +1,6 @@
 ## ---------------------------------------------------------------------
 ##
-## Copyright (C) 2006 - 2013 by the deal.II authors
+## Copyright (C) 2006 - 2013, 2015 by the deal.II authors
```

Comment:
Emacs did that! It will automatically be fixed next time we run the copyright script before the next release. But it's a valid point, I'll fix the issue.